### ISSUE TITLE
remove xtend

### ISSUE THREAD

**Issue by jimmywarting** [Level/level-ttl#128]
https://github.com/Level/level-ttl/blob/f7ead98e6b4631b4782be4fc787e25a61f4eda95/package.json#L20

remove xtend. [They deprecated xtend](https://github.com/Raynos/xtend#deprecated-xtend)

**Comment by vweevers** [comment]
Yes, we can. This fits well in https://github.com/Level/community/issues/98.

But: we're behind on maintaining `level-ttl`. It has race issues that need solving first.
